OurScript.R

```diff
@@ -0,0 +1,196 @@
+## Loading Necessary Libraries ----
+library(tidyverse)
+library(skimr)
+library(visdat)
+library(ggplot2)
+
+## Path to data (can be changed by user) 
+data_dir <- "data"
+target_file <- file.path(data_dir,"CrowdstormingDataJuly1st.csv")
+
+## Load data
+soccer_data <- read_csv(target_file)
+
+## Skim the data ----
+skim(soccer_data)
+
+#' The data shows that there are 28 variables, but the README file only mention 27. It appears that
+#' "ALPHA_3" is the additional variable on the dataset. It is a character variable with 160 unique abservations
+#' in it. 
+
+
+## Visualization of missing data
+# vis_miss(soccer_data,
+#          warn_large_data = FALSE)
+#' This is commented out because it takes a long time to run. Run at your own risk. :)
+
+## Visualization of smaller dataset
+set.seed(2020-10-22)
+soc_small <- sample_n(soccer_data, 15000)
```

Comment:
Good call creating this new dataset for viewing the data, keeping in mind the run times. Thank you!